calculator.c

```diff
@@ -0,0 +1,31 @@
+/*Matthew Danielson
```

Comment:
Overall, your assignment compiled, but it didn't run on all the needed cases. Your implementation was also not what was instructed. Be more careful with directions next time.